Source/Request/CMRequest.m

```diff
@@ -676,6 +676,13 @@ - (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticatio
 
 
 - (NSURLRequest *)connection:(NSURLConnection *)connection willSendRequest:(NSURLRequest *)request redirectResponse:(NSURLResponse *)response {
+	NSHTTPURLResponse *httpURLResponse = (NSHTTPURLResponse *)response;
+	if (httpURLResponse) {
+		if (httpURLResponse.statusCode >= kHTTPStatusMultipleChoices && httpURLResponse.statusCode < kHTTPStatusBadRequest) {
+			// it's a redirect
```

Comment:
I would prefer to see:
if (statusCode == HTTPMovedPermanently || statusCode == HTTPFound)

Was there a reason you chose this range?